naomi-setup.sh

```diff
@@ -7,54 +7,239 @@
 # install naomi & requirements in their
 # respective directories.
 #########################################
+OPTION="0"
+for var in "$@"; do
+    if [ "$var" = "--virtualenv" ]; then
+        OPTION="1"
+    fi
+    if [ "$var" = "--local" ]; then
+        OPTION="2"
+    fi
+    if [ "$var" = "--primary" ]; then
+        OPTION="3"
+    fi
+    if [ "$var" = "--help" ]; then
+        echo "USAGE: $0 [--virtualenv | --local | --primary | --help]"
+        echo
+        echo "  --virtualenv - install Naomi using a virtualenv environment for Naomi"
+        echo "                 (this is the recommended choice. You will need to issue"
+        echo "                 'workon Naomi' before installing additional libraries"
+        echo "                 for Naomi)"
+        echo
+        echo "  --local      - download, compile and install a special copy of Python 3"
+        echo "                 for Naomi (does not work for all distros)"
+        echo
+        echo "  --primary    - use your primary Python 3 environment"
+        echo "                 (this can be dangerous, it can lead to a broken python"
+        echo "                 environment on your system, which other software may"
+        echo "                 depend on. This is the simplest setup, but only recommended"
+        echo "                 when Naomi is the primary program you intend to run on this"
+        echo "                 computer. You will probably need to use the 'pip3' command"
+        echo "                 to install additional libraries for Naomi.)"
+        echo "                 USE AT YOUR OWN RISK!"
+        echo
+        echo "  --help       - Print this message and exit"
+        exit 0
+    fi
+done
+if [ $OPTION = "0" ]; then
+    echo 'There are three methods to install Naomi:'
```

Comment:
This can be hard to read due to the wrapping effect. You can see if you can echo to the user's terminal column. You can get the current terminal width with ```tput cols```. Default is 80 columns. 

https://www.unix.com/shell-programming-and-scripting/251927-limit-number-characters-bash-output.html 
This might help you...

Problem below
-----------------------
![image](https://user-images.githubusercontent.com/3073252/50799552-bce60c00-1299-11e9-97b4-d39ce44aa350.png)

Reply:
All lines now wrap to 80 characters.